src/towncrier/_version.py

```diff
@@ -7,5 +7,5 @@
 
 from incremental import Version
 
-__version__ = Version("towncrier", 19, 2, 0)
+__version__ = Version("towncrier", 21, 3, 0, dev=1)
```

Comment:
Maybe use dev4 as this was the last released ?

```suggestion
__version__ = Version("towncrier", 21, 3, 0, dev=4)
```

Reply:
Huh...  I definitely didn't mean to do this, lemme find where I lost that.  Thanks.

Reply:
Yeah, merge and fork and branch and PR and everything more confusion.  Sorry, thanks.  There was something else missing so this is a good catch.

https://github.com/twisted/towncrier/commit/149c27f5c45b030ed803accb5c2ce5539dd6e425